Plugins/ConnectSwiftPlugin/ConnectClientGenerator.swift

```diff
@@ -201,6 +207,30 @@ private extension MethodDescriptor {
         }
     }
 
+    func callbackAvailabilityAnnotation() -> String? {
```

Comment:
I think all these helper functions can be `private`

Reply:
I left them without explicit access to align with the functions that were already there; however, I think we're good on not needing to specify access level since the enclosing extension `private extension MethodDescriptor` is already marked.

Reply:
Ah didn't realize these were inside a private extension 👍🏽